src/App.vue

```diff
@@ -46,7 +46,7 @@
 					<v-col class="d-flex flex-row align-end justify-end">
 						<small class="pa-1" style="font-size: 0.6rem">
 							<code v-if="env !== 'production'" >{{ env }}</code>
-              <span v-else>Wikidown, made by <a href="https://github.com/trenz-gmbh/wikidown" target="_blank">TRENZ</a> with <span class="text-red">&hearts;</span></span>
```

Comment:
this should eventually be something more like:

    <span v-else>powered by<br>
    TRENZ<svg …>DOCS</span>

Reply:
See #11